### PR TITLE
fix(suite): correctly detect Android, Linux and Chrome OS users

### LINKED ISSUE #4380: Android is detected as Linux
**Describe the bug**
Android devices are detected as `Linux`. So `Linux` is being sent to analytics instead of `Android` and also `Android` users see tooltip with `Udev` rules.

_We also detect `ChromeOS` as `Android`_

### PR DESCRIPTION
**Pull request by tomasklim**
Closes #4380

`Android` was never detected correctly. It was detected as `Linux` because we checked `Linux` first and `navigator.platform` on `Android` is `Linux` https://stackoverflow.com/a/23736334

Moreover, a lot of people uses `Chrome OS` so we will start tracking it into analytics correctly. Now it is also tracked as `Linux`.

### CODE REVIEW

**Comment by tomasklim** [comment]
`IN PROGRESS` added, I will add some tests

File: packages/suite/src/utils/suite/env.ts [hunk 1]
```diff
@@ -41,32 +40,36 @@ export const isMacOs = () => {
...
+export const isChromeOs = () => /CrOS/.test(getUserAgent());
+
 export const getOsName = () => {
-    if (isMacOs()) return 'macos';
-    if (isLinux()) return 'linux';
     if (isWindows()) return 'windows';
+    if (isMacOs()) return 'macos';
     if (isAndroid()) return 'android';
+    if (isChromeOs()) return 'chromeos';
+    // Linux check has to be after Android and ChromeOS otherwise it would be incorrectly detected
```
Comment (matejkriz) [inline packages/suite/src/utils/suite/env.ts]: What about to add check for Android inside isLinux check? So the order doesn't matter then.

Comment (tomasklim) [reply packages/suite/src/utils/suite/env.ts]: 1380a18e847e7092134c4d2daed08a5cc36699ad

**Review by matejkriz — APPROVED** [review]
looks good, feel free to rebase fixup and merge

**Pull request merged**